imagepicker/src/main/java/com/esafirm/imagepicker/helper/LocaleManager.java

```diff
@@ -36,4 +37,30 @@ public static Context updateResources(Context context) {
         }
         return context;
     }
+
+    private static Locale zhCheck(Locale newLocaleLanguage) {
```

Comment:
Maybe change the function name so it can be more descriptive. 
`normalizeLocale` maybe?